lib/reply.js

```diff
@@ -681,7 +681,7 @@ function logStreamError (logger, err, res) {
 
 function sendWebStream (payload, res, reply) {
   if (payload.locked) {
-    throw FST_ERR_REP_READABLE_STREAM_LOCKED()
+    throw new FST_ERR_REP_READABLE_STREAM_LOCKED()
```

Comment:
`new` is unnecessary. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error